frontend/packages/kubevirt-plugin/src/components/vm-like/vm-like-details-page.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Firehose } from '@console/internal/components/utils';
+import { VirtualMachineInstanceModel, VirtualMachineModel } from '../../models';
+import { getResource } from '../../utils';
+import {
+  VirtualMachinesDetailsPage,
+  VirtualMachinesDetailsPageProps,
+} from '../vms/vm-details-page';
+
+export const VMLikeDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (props) => {
+  const { name, ns: namespace } = props.match.params;
+  const resources = [
+    getResource(VirtualMachineModel, {
```

Comment:
can we rather move this to VirtualMachinesDetailsPage and listen there on what is missing?

Reply:
done